### PR TITLE
[MIG] mgmtsystem_nonconformity: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- mgmtsystem_action: https://github.com/OCA/management-system/pull/366
- document_page_procedure: https://github.com/OCA/management-system/pull/369

### CODE REVIEW

File: mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        <field name="context">{"search_default_user_id":uid}</field>
+    </record>
+
+    <!-- <menuitem -->
+    <!--     id="menu_open_nonconformity" -->
+    <!--     string="Nonconformity" -->
+    <!--     action="open_mgmtsystem_nonconformity_list" -->
+    <!--     parent="mgmtsystem.menu_mgmtsystem_main" -->
+    <!--     sequence="30" -->
+    <!--     groups="mgmtsystem.group_mgmtsystem_manager,mgmtsystem.group_mgmtsystem_auditor,base.group_user"/> -->
```
Comment (michaelslade54) [inline mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml]: Unless I am missing something, without this menuitem, there is no way to access the nonconformity view from the menu. I think the 'string' attribute is causing an error when menuitem is uncommented.
```suggestion
    <menuitem
         id="menu_open_nonconformity"
         name="Nonconformities"
         action="open_mgmtsystem_nonconformity_list"
         parent="mgmtsystem.menu_mgmtsystem_main"
         sequence="30"
         groups="mgmtsystem.group_mgmtsystem_manager,mgmtsystem.group_mgmtsystem_auditor,base.group_user"/>
```

**Comment by dzungtran89** [comment]
indeed, thank you @michaelslade54 for your suggestion

File: mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+                                        widget="kanban_state_selection"
+                                    />
+                                    <img
+                                        t-att-src="kanban_image('res.users', 'image_small', record.responsible_user_id.raw_value)"
+                                        t-att-title="record.responsible_user_id.value"
+                                        t-att-alt="record.responsible_user_id.value"
+                                        width="24"
+                                        height="24"
+                                        class="oe_kanban_avatar"
+                                    />
```
Comment (michaelslade54) [inline mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml]:
```suggestion
                                    <field name="responsible_user_id" widget="many2one_avatar_user"/>
```

**Comment by YvanDotet** [comment]
Hello
Is it ready to merge in the 14.0 branch ?
Thank you.

File: mgmtsystem_nonconformity/readme/HISTORY.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+13.0.1.0.0 (2019-10-26)
```
Comment (simahawk) [inline mgmtsystem_nonconformity/readme/HISTORY.rst]: you can remove this (move to towcrier if you want)

File: mgmtsystem_nonconformity/readme/INSTALL.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+No installation required.
```
Comment (simahawk) [inline mgmtsystem_nonconformity/readme/INSTALL.rst]: you can remove this

File: test-requirements.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
```
Comment (simahawk) [inline test-requirements.txt]: drop these

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by etobella** [comment]
Can you squash all the migration commits please?

**Comment by dzungtran89** [comment]
> 
> 
> Can you squash all the migration commits please?

Hi @etobella , the PR has been updated

**Comment by simahawk** [comment]
@etobella good for you?

**Review by sebalix — APPROVED** [review]

**Pull request merged**